.github/workflows/build.yml

```diff
@@ -20,16 +20,8 @@ jobs:
       fail-fast: false
       matrix:
         major_version: [37, 38]
-        driver_version: [520, 525, 530, 470]
+        driver_version: [470, 530, 535]
         exclude:
-          - driver_version: 525
-            major_version: 37
-          - driver_version: 525
-            major_version: 38
-          - driver_version: 520
-            major_version: 37
-          - driver_version: 520
-            major_version: 38
           - driver_version: 470
```

Comment:
Exclude builds of 37 with driver version 530 here
```
- driver_version: 530
  major_version: 37
```

Reply:
If you still wants some legacy in your version you can keep the 520 driver on fedora 37 cause it's the base driver so it will always be possible to build it.

Reply:
Changes made in 182798f

I don't think we need to worry about enabling an older driver version in this PR - this was mainly just getting the new ones to build

Reply:
Sure that's fine to keep the latest only.

I was just saying this so you know what's possible to do or not in the current situation it might help understanding for the future so that kind of issue won't happen any more.